nsxt/resource_nsxt_policy_ipsec_vpn_session.go

```diff
@@ -0,0 +1,482 @@
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"vpn_type": {
+				Type:         schema.TypeString,
+				Description:  "A Policy Based VPN requires to define protect rules that match local and peer subnets. IPSec security associations is negotiated for each pair of local and peer subnet. A Route Based VPN is more flexible, more powerful and recommended over policy based VPN. IP Tunnel port is created and all traffic routed via tunnel port is protected. Routes can be configured statically or can be learned through BGP. A route based VPN is must for establishing redundant VPN session to remote site.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionResourceType, false),
+				Optional:     true,
+			},
+			"compliance_suite": {
+				Type:         schema.TypeString,
+				Description:  "Connection initiation mode used by local endpoint to establish ike connection with peer site. INITIATOR - In this mode local endpoint initiates tunnel setup and will also respond to incoming tunnel setup requests from peer gateway. RESPOND_ONLY - In this mode, local endpoint shall only respond to incoming tunnel setup requests. It shall not initiate the tunnel setup. ON_DEMAND - In this mode local endpoint will initiate tunnel creation once first packet matching the policy rule is received and will also respond to incoming initiation request.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionComplianceSuite, false),
+				Optional:     true,
+			},
+			"connection_initiation_mode": {
+				Type:         schema.TypeString,
+				Description:  "Compliance suite.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionConnectionInitiationMode, false),
+				Optional:     true,
+				Default:      "INITIATOR",
+			},
+			"authentication_mode": {
+				Type:         schema.TypeString,
+				Description:  "Peer authentication mode. PSK - In this mode a secret key shared between local and peer sites is to be used for authentication. The secret key can be a string with a maximum length of 128 characters. CERTIFICATE - In this mode a certificate defined at the global level is to be used for authentication.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionAuthenticationMode, false),
+				Optional:     true,
+				Default:      "PSK",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Description: "Enable/Disable IPSec VPN session.",
+				Optional:    true,
+				Default:     true,
+			},
+			"dpd_profile_path": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing Dead Peer Detection (DPD) profile. Default is set to system default profile.",
+				Optional:    true,
+				Default:     "/infra/ipsec-vpn-dpd-profiles/nsx-default-l3vpn-dpd-profile",
```

Comment:
We don't use NSX absolute paths in the provider. Is profile path required or default is taken if not provided?